2021/Miglior/README.md

```diff
@@ -0,0 +1,47 @@
+## Using candlestick charts to identify network anomalies in timeseries data
+
+The aim of this project is to identify anomalies and outliers inside a time series, using candlestick charts.
+The forecasting algorithm has been based on (double) exponential moving average.
+Monitoring the three main candle values (open value, close value, and volume) allows us to find anomalies even in
```

Comment:
Nella serie temporale fornita in formato rrd, non c'e' il concetto di open/close/volume: devi spiegare come sono definiti e come si applicano a serie temporali con valori contenenti dati numerici di rete.